saagieapi/jobs/jobs.py

```diff
@@ -33,8 +34,10 @@ def list_for_project(self, project_id: str, instances_limit: int = -1) -> Dict:
         params = {"projectId": project_id}
         if instances_limit != -1:
             params["instancesLimit"] = instances_limit
-        query = gql(GQL_LIST_JOBS_FOR_PROJECT)
-        return self.saagie_api.client.execute(query, variable_values=params)
+
+        return self.saagie_api.client.execute(
+            query=gql(GQL_LIST_JOBS_FOR_PROJECT), variable_values=params, pprint_result=True
```

Comment:
Add `pprint_result` as parameter ?

Reply:
not needed as this function is not called by other functions